src/map/lapack2flash/FLA_lapack2flame_prototypes.h

```diff
@@ -0,0 +1,1743 @@
+/*
+
+    Copyright (C) 2014, The University of Texas at Austin
```

Comment:
maybe this is a good location to get the generic block size / #threads set/get in?

Reply:
As I understand it, these are all of the lapack calls. I had my eyes on FLASH_lapack2flash_util.c for the get/sets unless you think this would be better.

Reply:
See here: https://github.com/iotamudelta/libflame/pull/2/commits/926e05ee52bc9f34e87d35af34eb71042ab6b6ed

I went with FLASH_lapack2flash_util.c, but can easily change this if here would be better.

Reply:
good!